daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -1347,35 +1347,12 @@ Safe strncpy that ensures a trailing zero
 =============
 */
 
-#ifndef NDEBUG
```

Comment:
You can probably remove it since moving to std::string has more or less deprecated this function. There used to be a define that would #define Q_strncpyz( src, dst, size ) to Q_strncpyzDebug( src, dst, size, __FILE__, __LINE__ ) or something and it would be used engine wide and help track down null src/dst bugs.